roles/nginxplus/files/conf/http/lib-statistics-prod.conf

```diff
@@ -0,0 +1,42 @@
+# Ansible managed
+proxy_cache_path /data/nginx/lib-statistics-prod/NGINX_cache/ keys_zone=lib-statistics-prodcache:10m;
+
+upstream libstatistics-prod {
+    zone libstatistics-prod 64k;
+    server lib-statistics-prod1.princeton.edu resolve;
+    sticky learn
+          create=$upstream_cookie_lib-statistics-prodcookie
+          lookup=$cookie_lib-statistics-prodcookie
+          zone=lib-statistics-prodclient_sessions:1m;
+}
+
+server {
+    listen 80;
+    server_name lib-statistics.princeton.edu;
+
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl http2;
+    server_name lib-statistics.princeton.edu;
+
+    ssl_certificate            /etc/nginx/conf.d/ssl/certs/lib-statistics_princeton_edu_chained.pem;
+    ssl_certificate_key        /etc/nginx/conf.d/ssl/private/lib-statistics_princeton_edu_priv.key;
+    ssl_session_cache          shared:SSL:1m;
+    ssl_prefer_server_ciphers  on;
+
+    location / {
+        proxy_pass http://lib-statistics;
```

Comment:
this wants to point to the upstream on line 4 (so you want the name to be `http://libstatistics-prod`

Reply:
Thanks!